lms/djangoapps/bookmarks/urls.py

```diff
@@ -0,0 +1,27 @@
+"""
+Unit Bookmark API URLs
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from .views import BookmarksView, BookmarksDetailView
+
+USERNAME_PATTERN = r'(?P<username>[\w.@+-]+)'
+
+urlpatterns = patterns(
+    "bookmarks",
+    url(
+        r"^v0/bookmarks/$",
+        BookmarksView.as_view(),
+        name="bookmarks"
+    ),
+    url(
+        r"^v0/bookmarks/{username},{usage_key}/$".format(
```

Comment:
I thought the API specs things are final. All right, doing it now, We should also update the API spec accordingly.